config.go

```diff
@@ -93,7 +95,18 @@ func (config Config) readYmlFile(file string) *Config {
 }
 
 func (config Config) parse(yml []byte) *Config {
+	envMap := envToMap()
+	configTemplate := template.New("config")
+	configTemplate, _ = configTemplate.Parse(string(yml[:]))
+	var configTpl bytes.Buffer
+	renderingErr := configTemplate.Execute(&configTpl, envMap)
+	if renderingErr != nil {
```

Comment:
if there is a rendering error config should panic

Reply:
do not use fmt.println

Reply:
instead put the reason for the rendering error into the config.panic() message so bootstrap aborts